ChatApplication/Startup.cs

```diff
@@ -82,5 +99,19 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 endpoints.MapRazorPages();
             });
         }
+
+        public async Task AddDbDateCache(IServiceCollection services)
+        {
+            var cacheKey = "Allmessages0";
```

Comment:
you have hardcoded the same string multiple times. declare the cache key as constant somewhere and use constant instead of hardcoding